gef.py

```diff
@@ -3766,15 +3766,38 @@ def dereference(addr: int) -> Optional["gdb.Value"]:
     return None
 
 
-def gef_convenience(value: str) -> str:
-    """Defines a new convenience value."""
+def gef_gen_convenience_name() -> str:
+    """Defines a new convenience name."""
     global gef
     var_name = f"$_gef{gef.session.convenience_vars_index:d}"
     gef.session.convenience_vars_index += 1
+    return var_name
+
+
+def gef_convenience(value: str) -> str:
+    """Defines a new convenience value."""
+    global gef
+    var_name = gef_gen_convenience_name()
     gdb.execute(f"""set {var_name} = "{value}" """)
     return var_name
 
 
+def gef_conveniencebt(value: bytes) -> str:
```

Comment:
This function is not strictly necessary. You can let Python do the heavy lifting as such:

```python
def gef_convenience(value: Union[str, bytes]) -> str:
    """Defines a new convenience value."""
    global gef
    var_name = gef_gen_convenience_name()
    if isinstance(value, str:)
        gdb.execute(f"""set {var_name} = "{value}" """)
    elif isinstance(value, bytes):
        value_as_array = "{" + ", ".join(["%#.02x" % x for x in value]) + "}"
        gdb.execute(f"""set {var_name} = {value_as_array} """)
    else:
        raise TypeError
    return var_name
```

Reply:
done!